README.md

```diff
@@ -59,8 +59,8 @@ To return to the Welcome screen, simply unselect the item by holding Control whi
 ### Remote Processes
 The list on the bottom left shows any remote processes.
 Connections to remote processes can be established either by using the *+ button* to the right of the heading, or by going to *Connect → New Connection*.
-### Local FS (file-system)
-Here you can turn runtime decompiler to casual, user friendly jar/folder/file decompiler. You create FS vm by setting up classpath as you are used for JVM.
+### Local FS (file system)
+Here you can turn JRD to casual, user-friendly jar/folder/file decompiler. You create a FS VM by setting up classpath as you are used for JVM.
```

Comment:
the shortcut is nowhere deffined. Maybe to do so on line 1?

Reply:
Addressed in 10b43b3.